File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/typescript/ADV-06_R.js
"use strict";
console.log("ADV-START");
const lines = require("fs").readFileSync(0, "utf8").split("\n");
const a = Number.parseFloat(lines[0]);
const b = Number.parseFloat(lines[1]);
const m = a / b;
const seq = [3.0, m, 1.0];
let best = seq[0];
for (let i = 1; i < seq.length; i++) {
    if (seq[i] > best) {
        best = seq[i];
    }
}
const selfeq = m === m;
console.log("OBS=MAX:" + best.toFixed(6) + "|SELFEQ:" + selfeq);
console.log("ADV-END");

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/swift/ADV-06_R.swift
import Foundation
func main() {
    print("ADV-START"); fflush(stdout)
    let a = Double(readLine()!)!
    let b = Double(readLine()!)!
    let m = a / b
    let xs: [Double] = [3.0, m, 1.0]
    var best = xs[0]
    for x in xs[1...] {
        if x > best { best = x }
    }
    let selfeq = (m == m)
    print("OBS=MAX:\(String(format: "%.6f", best))|SELFEQ:\(selfeq)"); fflush(stdout)
    print("ADV-END"); fflush(stdout)
}
main()
